<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGameWindow</class>
 <widget class="QMainWindow" name="MainGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>迷之连连看</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/res/image/background/5.jpg);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="timeBar">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>11</y>
      <width>561</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>39</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>时间：</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>652</x>
      <y>10</y>
      <width>401</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="hintBtn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(11, 255, 36, 161);</string>
       </property>
       <property name="text">
        <string>提示</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pause_btn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(11, 255, 36, 161);</string>
       </property>
       <property name="text">
        <string>暂停游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="robot_btn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(11, 255, 36, 161);</string>
       </property>
       <property name="text">
        <string>作弊</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restart_btn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(11, 255, 36, 161);</string>
       </property>
       <property name="text">
        <string>重新开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quit_btn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(11, 255, 36, 161);</string>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>60</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(11, 255, 36, 161);</string>
    </property>
    <property name="text">
     <string>重新排列</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>游戏模式(&amp;M)</string>
    </property>
    <addaction name="actionBasic"/>
    <addaction name="actionMedium"/>
    <addaction name="actionHard"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <action name="actionBasic">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/image/19.jpg</normaloff>:/res/image/19.jpg</iconset>
   </property>
   <property name="text">
    <string>basic</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/image/4.jpg</normaloff>:/res/image/4.jpg</iconset>
   </property>
   <property name="text">
    <string>medium</string>
   </property>
  </action>
  <action name="actionHard">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/image/62.jpg</normaloff>:/res/image/62.jpg</iconset>
   </property>
   <property name="text">
    <string>hard</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quit_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainGameWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>921</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>991</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
